Common/Core/trackUtilities.h

```diff
@@ -128,4 +130,23 @@ void getPxPyPz(T const& trackPars, U& pVec)
   pVec[2] = pt * trackPars.getTgl();
 }
 
-#endif // O2_ANALYSIS_TRACKUTILITIES_H_
+/// Calculates DCA xyz and if uncertainty is required, return its value by pointer
+/// To use this function, use the o2::aod::TracksWCovDca table
+/// \parma track are the track parametersd which are included dca information
+/// \param sigmaDcaXYZ2 is impact parameter sigma^2 in XYZ of the track to the primary vertex
+/// \return dcaXYZ is impact parameter in XYZ of the track to the primary vertex
+template <typename T>
+float getDcaXYZ(T const& track, float* sigmaDcaXYZ2 = nullptr)
+{
+  float dcaXY = track.dcaXY();
+  float dcaZ = track.dcaZ();
+  float dcaXYZ = std::sqrt(dcaXY * dcaXY + dcaZ * dcaZ);
+  if (sigmaDcaXYZ2) {
+    float dFdxy = 2 * dcaXY / dcaXYZ;
+    float dFdz = 2 * dcaZ / dcaXYZ;
```

Comment:
A protection against division by zero might be useful.

Reply:
I've fixed protection and description of function.
Please confirm that again.